File: java/05-printfile/printfile.java
import java.util.Scanner;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;

public class printfile
{
    public static void main(String[] args)
    {
  	Scanner scanner = new Scanner(System.in);

	System.out.println("Enter the complete path to a file:");
	String pathstr = scanner.nextLine();
	Path path = Path.of(pathstr);

	try {
	    String text = Files.readString(path);
	    System.out.println("The size of that file is: " + text.length());
	    System.out.println("Read in:\n" + text);
	}
	catch  (IOException e) {
	    System.out.println("An exception occured");
	    System.out.println(e.getMessage());
	}
    }
}

File: python/05-printfile/printfile.py
path = input("Enter the complete path to a file: ")

#var2 = int(input("Enter another number: "))
#print("The sum is {}".format(var1+var2))

with open(path) as f:
    text = f.read()

print("The size of that file is", len(text))

print("Read in:\n{}".format(text))
